ts/Base/MaterialComponent.ts

```diff
@@ -76,9 +76,11 @@ export abstract class MaterialComponent<
     }
 
     const element = this.materialDom(componentProps);
-    element.attributes = element.attributes || {};
+    // @ts-ignore
```

Comment:
Are the types bad? `@ts-ignore` should be avoided if necessary.

Reply:
[The types have `props` defined](https://github.com/preactjs/preact/blob/4a581a1c4750ca5d71e416deaa3bdf3061b9dbe5/src/index.d.ts#L15), are you sure, there aren't bad types lingering around?